spec/text_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'text'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.